packages/lab/src/parent-child-layout/ParentChildLayout.css

```diff
@@ -1,24 +1,28 @@
 .saltParentChildLayout {
-  overflow: hidden;
+  display: flex;
 }
 
-.saltParentChildLayout-parent {
-  animation: var(--salt-animation-slide-in-left);
+.saltParentChildLayout-child {
+  flex-grow: 1
 }
 
-.saltParentChildLayout-child {
-  animation: var(--salt-animation-slide-in-right);
+
+.saltParentChildLayout-collapsed {
+  width: 100%;
```

Comment:
When collapsed the parent / child should take up 100% of the parent child layout